tests/scripts/run-xunit-perf.py

```diff
@@ -149,7 +149,7 @@ def run_command(runArgs, environment, errorMessage):
         subprocess.check_output(runArgs, stderr=subprocess.PIPE, env=environment)
     except subprocess.CalledProcessError as e:
         log(e.output.decode('utf-8'))
```

Comment:
Should we be doing this:
```python
except subprocess.CalledProcessError as e: 
        log(errorMessage)  # errorMessage(str): message to print if there is an error
        log(e.output.decode('utf-8')) 
        raise e
```

Reply:
I can, if you'd prefer it that way.

Reply:
Yes, please. Let's do this (Bubble up the caught exception).

Reply:
done